torch/_C/_distributed_c10d.pyi

```diff
@@ -18,10 +18,11 @@ def _register_builtin_comm_hook(reducer: Reducer, comm_hook_type: BuiltinCommHoo
 class _GradBucket:
     def __init__(self, tensors: List[Tensor]): ...
     def get_index(self) -> int: ...
-    def get_lengths(self) -> List[int]: ...
+    def get_tensors(self) -> List[Tensor]: ...
+    def get_per_parameter_tensors(self) -> List[Tensor]: ...
     def get_offsets(self) -> List[int]: ...
+    def get_lengths(self) -> List[int]: ...
```

Comment:
Are `get_offsets`, `get_sizes`, and `get_lengths` no longer intended to be used by GradBucket class user? If so, consider making those private APIs?

Reply:
You are right.

Actually I removed them in a follow-up PR. I don't think we will use them.